packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -242,7 +275,9 @@ export function getLocalDataStoreType(localDataStore: LocalFluidDataStoreContext
  * but eventually could be hosted on any channel once we formalize the channel api boundary.
  * @internal
  */
-export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
+export class ChannelCollection
+	implements Omit<IFluidDataStoreChannel, "entryPoint" | "reSubmit" | "rollback">, IDisposable
```

Comment:
I'm a little wary of this. How will it affect nested datastores? I thought implementing `IFluidDataStoreChannel` was _the_ key feature of this thing, for the purposes of nested datastores.

Reply:
Ah, I see `ComposableChannelCollection` below.  Never mind